### ISSUE TITLE
Dynamic resources folder

### ISSUE THREAD

**Issue by estudiooka** [LIN3S/WPRouting#5]
The possibility to change the resources folder name and or location without hack the package would be great. Maybe a filter or a config variable.

**Comment by benatespina** [comment]
Yes, that would be great! We always use that path but in case you come up with a way to make it more flexible we will be more than happy to merge a PR.

Thanks!!
